zgw/notificaties-api/src/main/kotlin/com/ritense/notificatiesapi/security/config/NotificatiesApiHttpSecurityConfigurer.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.notificatiesapi.security.config
+
+import com.ritense.valtimo.contract.security.config.HttpConfigurerConfigurationException
+import com.ritense.valtimo.contract.security.config.HttpSecurityConfigurer
+import org.springframework.http.HttpMethod
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+
+class NotificatiesApiHttpSecurityConfigurer : HttpSecurityConfigurer {
+    override fun configure(http: HttpSecurity) {
+        try {
+            http.authorizeRequests()
+                .antMatchers(HttpMethod.POST, "/api/v1/notificaties/send")
```

Comment:
Not sure about the `/send`. Since it doesn't really send anything. It only receives notifications

Reply:
The `/send` is still there

Reply:
while testing locally, I made a temporary one with this path: `/api/v1/notificatiesapi/notification`. it made sense to me that we keep the path consistent with the plugin name. would this be an option?

Reply:
@Klaas-Ritense it can't be i removed it.
@anna-ritense can we compromise on "/api/v1/notificaties/notification"? my reasoning being that "notificatiesapi" is not a resource that's being targeted. just "notificaties" seems more like the resource itself.

Reply:
I vote for the second one, that way the resource is clearly different from the old one that's provided by the connector.

Reply:
Luis, can you also change the url in the `NotificatiesApiHttpSecurityConfigurer.kt`